### ISSUE TITLE
repo init permissions bug?

### ISSUE THREAD

**Issue by jbenet** [ipfs/go-ipfs#1273]
https://travis-ci.org/ipfs/go-ipfs/jobs/63442607

```
--- init_fail_out	2015-05-21 07:04:12.000000000 +0000
+++ init_fail_exp	2015-05-21 07:04:12.000000000 +0000
@@ -1,2 +1 @@
-Error: ipfs daemon is running. please stop it to run this command
-Use 'ipfs init --help' for information about this command
+Error: open /Users/travis/gopath/src/github.com/ipfs/go-ipfs/test/sharness/trash directory.t0020-init.sh/.badipfs/repo.lock: permission denied
not ok 3 - ipfs init output looks goodm
#	
#		echo "Error: open $IPFS_PATH/repo.lock: permission denied" > init_fail_exp &&
#		test_cmp init_fail_out init_fail_exp
#	
e
```

**Comment by whyrusleeping** [comment]
we only see this occasionally... not sure what would cause it to think a daemon is running like this

**Comment by whyrusleeping** [comment]
being addressed in #1276
